### ISSUE TITLE
renderer: anonymous non-C-compatible type given name for linkage purposes by typedef declaration

### ISSUE THREAD

**Issue by davidebeatrici** [RobertBeckebans/RBDOOM-3-BFG#643]
```cpp
neo/idlib/./../renderer/Material.h:236:15: warning: anonymous non-C-compatible type given name for linkage purposes by typedef declaration; add a tag name here [-Wnon-c-typedef-for-linkage]
typedef struct
              ^
               stencilStage_t
neo/idlib/./../renderer/Material.h:240:13: note: type is not C-compatible due to this default member initializer
        byte ref = 0;
                   ^
neo/idlib/./../renderer/Material.h:262:3: note: type is given name 'stencilStage_t' for linkage purposes by this typedef declaration
} stencilStage_t;
  ^
```

**Comment by SRSaunders** [comment]
This one is solved by my latest pull request #645.  All it needed was a tag name to be added.

**Comment by davidebeatrici** [comment]
Indeed, thanks!

**Comment by SRSaunders** [comment]
This should probably be closed since it has been subsequently fixed in master (see commit id 1d62816).
